Week3/homework/step2-1.js

```diff
@@ -4,6 +4,7 @@ function foo(func) {
   // What to do here?
   // Replace this comment and the next line with your code
   console.log(func);
+  return func();
```

Comment:
You are right that you should call `func` (which is, in this case, is a reference to `bar`. It is not necessary to return the return value of `func/bar` (``bar` doesn't return anything and therefore implicitly returns `undefined`):

```js

function foo(func) {
  func();
}
```